### PR TITLE
Merge 2.19 final to trunk

### PR DESCRIPTION
**Pull request by oguzkocer**

### CODE REVIEW

**Comment by AliSoftware** [comment]
⚠️ @oguzkocer wrong target branch for this PR?

**Comment by oguzkocer** [comment]
@AliSoftware Yes, sorry 🤦 Should be fixed now.

File: Simplenote/build.gradle
Comment (AliSoftware) [inline Simplenote/build.gradle:35]: Unsure why the jump, but 🤷

File: fastlane/Pluginfile
Comment (AliSoftware) [inline fastlane/Pluginfile:5]: Side note: you might want to update to `~> 1.4` (and _not_ `~> 1.4.0`, btw) in `develop` at some point during next sprint 🙂

**Review by AliSoftware — APPROVED** [review]
Branch graph makes sense and version bumps seems good too, ready to ship!

~_(not sure if you prefer waiting for Google approval on monday to merge or not, so will let you push the button)_~ eh you enabled auto-merge after all 👍

**Pull request merged**
